code/game/mecha/combat/five_stars.dm

```diff
@@ -0,0 +1,22 @@
+/obj/mecha/combat/five_stars
```

Comment:
why is this a mecha

Reply:
Because mechas do everything I want this to do?

Reply:
it's not really the same as a mech in a lot of its behavior.

Reply:
>is a object with weapons attached 
>can be piloted
>has a health system

Mecha code does everything I want this to do.

Reply:
surely my fighter plane code would also fit perfectly into /obj/mecha seeing as it also does those 3 things.

There's a ton of vars/procs on mechs that have nothing to do with tanks. Don't make the heap of crap thats mecha code more unmaintainable

Reply:
How the fuck does this make mecha code more unmaintainable? It's a single typepath. I'm not fucking homebrewing a carbon copy of fucking mecha code when mecha code does what I want AND has the added bonus of "if they really want to they can go to robotics and print bonus weapons for it".

Making a new fucking framework is out of goddamn scope for a gamemode PR and you know it, dude.

Reply:
you can't say it's out of scope and add a tank in the same PR. Make the tank a seperate PR if you're not willing to do it properly.

Reply:
It is literally fine using mecha code. I'll fix some of the weirder stuff but it's staying as a mecha unless an actual maintainer says otherwise.